src/main/java/com/thebois/controllers/game/StructureController.java

```diff
@@ -62,6 +62,7 @@ public IView getView() {
 
     @Override
     public void update() {
+        structureView.update(world.getStructures());
```

Comment:
Seems like I accidentally added it because I didn't find where it was done when I changed the getStructures method.

Originally it was done on touch down, but I'm not sure it should? I can remove it, either way works.

Reply:
Perhaps, depends on if we want to send a cloned list or not. Otherwise the View already has a reference to all the houses